### ISSUE TITLE
No longer able to use scroll wheel to adjust volume over miniplayer since last update

### ISSUE THREAD

**Issue by Popinaodude** [MarshallOfSound/Google-Play-Music-Desktop-Player-UNOFFICIAL-#1428]
Like the title says, no longer able to use scroll wheel on mouse to adjust volume on miniplayer. Now have to open full window to adjust volume.

I realise this is a really tiny problem but I thought being able to use the scroll wheel to adjust the volume of the player without having to open anything was super convenient. Thank you for your time!

**Comment by wefacd2wedcs** [comment]
I opened a ticket like this earlier today but he told me I was doing it wrong, closed the ticket, and ignored my replies that I wasn't doing anything wrong.

 Good luck. Maybe we can get it fixed.

**Comment by notasmartman** [comment]
If you go to the desktop settings, then miniplayer, there is a checkbox to use the scrollwheel to adjust volume. I assume the default is unchecked.

**Comment by Popinaodude** [comment]
You're right there is a checkbox I had totally forgotten about that. Strange that the update unchecked it.
Thank you notasmartman you are clearly a smarter man than me

**Comment by MarshallOfSound** [comment]
@wefacd2wedcs I didn't ignore your replies... I went to sleep....

Be patient, I wait for response for weeks and handle issues within minutes normally.  Don't expect me to be sitting at my computer dedicated to solving your issues because I will simply ignore you in the future.

As for this issue as @notasmartman mentioned there is now a Desktop Setting for scrolling and it may be off by default due to a volume issue people were having (better safe than sorry).  If that settings fixed your issue @Popinaodude please close this issue.  Otherwise comment again and I'll look further

**Comment by Popinaodude** [comment]
Actually I have restarted my computer and everything and I am still having the problem. The checkbox is ticked but scrolling to change the volume does not work on the miniplayer

**Comment by jostrander** [comment]
Confirmed. Testing further.

**Comment by MarshallOfSound** [comment]
@jostrander Appears to be an issue with GMusic.js

```
Uncaught AssertionError {name: "AssertionError", actual: false, expected: true, operator: "==", message: "Expected target volume (NaN) to be >= 0"…}
```

**Comment by MarshallOfSound** [comment]
Problem isolated, fix going upstream.

**Comment by wefacd2wedcs** [comment]
@MarshallOfSound I get the sleep deal. I was just cranky because you closed my ticket without giving me the chance to show you that what you said to do didn't work and in fact, you had told me to do it another way in the past. Plus the issue with next/previous buttons on the mouse not working anymore was ignored.

So in the end, you had to have a duplicate ticket submitted by someone else before actually listening. If you ignore me in the future, that's fine but it's not like I got anywhere with my previous ticket.

**Comment by MarshallOfSound** [comment]
@wefacd2wedcs To put in in perspective mate, I get probably around 20 issues a day raised on this repository alone (not counter the other repositories I maintain or help maintain). I made a call that it was a non-issue and moved on with life.  I would have reopened your issue the next morning if the second issues hadn't been raised and been more active.

The next / previous wasn't ignored.  It had already been fixed a few hours previously.

TLDR: Don't think just because I close an issue I am done with it and definitely don't think that because I need to sleep I'm ignoring you...
